pikaur/print_department.py

```diff
@@ -614,4 +629,17 @@ def get_aur_sort_key(pkg: AURPackageInfo) -> Tuple[float, str]:
                 rating
             ))
             print(format_paragraph(f'{package.desc}'))
+
+            if (
+                user_config.ui.AurSearchSorting == 'lastmodified'
+            ) and (
+                isinstance(package, AURPackageInfo)
+            ) and (
+                package.lastmodified is not None
+            ):
+                last_modified_date = "(last updated: {})".format(
+                    datetime.fromtimestamp(package.lastmodified).strftime('%Y/%m/%d')
+                )
+                print(format_paragraph(color_line(f'{last_modified_date}', 8)))
```

Comment:
it's printing it on the new line?

Reply:
what about the end of the first line (but in the same color as now)?

Reply:
This is what I currently have (I added the "last updated:" string just now):
![image](https://user-images.githubusercontent.com/8577904/97696761-25028380-1afa-11eb-9149-fcae1201774d.png)

After editing and trying your suggestion, this is what it currently looks like:
![image](https://user-images.githubusercontent.com/8577904/97697245-d73a4b00-1afa-11eb-9e01-0ed6a9ee3439.png)

After a second thought, I think the second one might look better as I made my terminal wider. Let me know your preference.

Reply:
i'm just thinking from the point of uniformity: since that additional field (last updated) is shown as a passive side-effect of another config options, it wouldn't be nice changing UI output too much.

alternatively, you could implement additional option for showing last updated date in search results (not depending on sorting) - in that case that would look reasonable

Reply:
Right, I added it because they might want to know the last updated that if they sort it that way. I agree with you about uniformity though - it does look a bit weak on that point.

I reckon I can just remove the date display entirely and see if a future issue pops up that requests this? If say we want to add an option as you said, then I'd need to do the same for official packages and those are structured a bit differently.

Reply:
for repo packages field called "Build Date" (which technically different, but from the user perspective not much)

Reply:
Okay, I'll separate that feature as an extra option tonight

Reply:
thanks for understanding :)